src/services/logger/logger.ts

```diff
@@ -0,0 +1,34 @@
+import log from 'loglevel';
+
+const defaultLevel = process.env.NODE_ENV === 'production' ? 'info' : 'trace';
+log.setDefaultLevel(defaultLevel);
+
+const trace = (...msg: any[]) => {
+  log.trace(...msg);
+};
+
+const debug = (...msg: any[]) => {
+  log.debug(...msg);
+};
+
+const info = (...msg: any[]) => {
+  log.info(...msg);
+};
+
+const warn = (...msg: any[]) => {
+  log.warn(...msg);
+};
+
+const error = (...msg: any[]) => {
+  log.error(...msg);
+};
+
+const logger = {
+  trace,
+  debug,
+  info,
+  warn,
+  error,
+};
```

Comment:
[non-blocking] you could potentially do, to save lines and preserve types better: 

```
const logger = {
  trace: log.trace,
etc
}
```